src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export function Person({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{person.name}</h2>
+      {person.age ? (
+        <p className="Person__age">
+          I am
+          {' '}
+          {person.age}
+        </p>
+      ) : <p>{undefined}</p>
+      }
+
+      {person.isMarried ? (
+        <p className="Person__partner">
+          My
+          {
+            person.isMarried && person.sex === 'm'
+              ? (' wife')
+              : (' husband')
+          }
+          &apos;s name is
+          {' '}
+          {person.partnerName}
+        </p>
+      ) : <p>I am not married</p>
+      }
```

Comment:
too complicated
Will be better to create separate variable before return statement in component